src/pages/Program/Program.css

```
@@ -127,4 +127,51 @@ div#programmeContent div.rokyContainer {
+div#programmeContent label.rokKarta:hover .rokFront {
+    color: var(--mint);
+}
+
+
+.nadrokovybox{
+    background-color: var(--asociální-zeleň);
+  border: 3px solid var(--mint);
+  border-radius: 10px;
+  color: white;
+  font-family: monospace, monospace;
+  font-weight: 600;
```

Biome (2.1.2)

[error] Duplicate font names are redundant and unnecessary: monospace

Remove duplicate font names within the property

(lint/suspicious/noDuplicateFontNames)